tensorboard/components/tf_storage/storage.ts

```diff
@@ -54,6 +54,11 @@ export let TAB = '__tab__';
 export let DISAMBIGUATOR = 'disambiguator';
 
 /**
+ * The (string) name for the run of the selected dataset in the graph dashboard.
+ */
+export const SELECTED_DATASET = 'selected_dataset';
```

Comment:
Why isn't this just called `'run'`? We call it a run everywhere else in the UI (within this plugin and across other plugins); I don't see the need for new terminology.